Classes/Controller/OaiPmhController.php

```diff
@@ -714,7 +715,7 @@ protected function generateOutputForDocumentList(array $documentListSet)
         }
         $verb = $this->parameters['verb'];
 
-        $documents = $this->documentRepository->getOaiDocumentList($this->settings, $documentsToProcess);
+        $documents = $this->documentRepository->getOaiDocumentList($documentsToProcess);
```

Comment:
Is this change intended or some kind of merging error?

Reply:
Not sure how it has landed here but looks like correct thing to do considering that `public function getOaiDocumentList(array $documentsToProcess): DriverResultStatement` doesn't take two parameters.